CI/travis.linux.before_install.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 set -ev
+
+exit 0
```

Comment:
Does this mean - bail out here unconditionally and successfully - thereby skipping the remainder of the file - if so, why not just delete the rest?